quickstart_template/301-jenkins-aptly-spinnaker-vmss.sh

```diff
@@ -47,10 +46,14 @@ function run_util_script() {
 
 # Set defaults
 region="westus"
+clusterName="aks101cluster"
 repository_name="hello-karyon-rxnetty"
-artifacts_location="https://raw.githubusercontent.com/Azure/azure-devops-utils/master/"
+artifacts_location="https://raw.githubusercontent.com/onlyloveyouever/azure-devops-utils/master/"
 artifacts_location_sas_token=""
-front50_port="8081"
+front50_port="8080"
+app_id="591d345d-ce5d-4368-8442-07fbb9d93e26"
```

Comment:
These variables should be put in the parameters like `storage_account_key`. And do not upload test data to GitHub.